doc/Comments.xml

```diff
@@ -343,6 +343,10 @@ use @ExampleSession:
 ]]></Listing>
 It inserts an example into the manual just as @Example would do, but all lines are commented
 and therefore not executed when the file is read.
+All lines have to start with an &AutoDoc; comment
+(<C>#!</C>). The comment and the following character are removed for the manual.
```

Comment:
So... we now accept everything? Even `#!TextWithNoSpace`, and then truncate that to `extWithNoSpace`?

Reply:
Yes. How should we decide what is the right behaviour? Only remove spaces? We could also do this.